ansible/roles/test/files/ptftests/sflow_test.py

```diff
@@ -193,14 +193,14 @@ def analyze_counter_sample(self, data, collector, polling_int, port_sample):
 
     def analyze_flow_sample(self, data, collector): 
         logging.info("packets collected from interfaces ifindex : %s" %data['flow_port_count'])
-        logging.info("Expected number of packets from each port : %s to %s"%(50*0.6,50*1.4))
+        logging.info("Expected number of packets from each port : %s to %s"%(100*0.6,100*1.4))
         for port in self.interfaces:
             ifindex = self.interfaces[port]['ifindex'] 
             logging.info("....%s : Flow packets collected from port %s = %s"%(collector,port,data['flow_port_count'][ifindex]))
             if port in self.enabled_intf :
                 # Checking samples with tolerance of 40 % as the sampling is random and  not deterministic.Over many samples it should converge to a mean of 1:N
-                # Number of packets sent = 50 * sampling rate of interface 
-                self.assertTrue(50*0.6 <= data['flow_port_count'][ifindex] <= 50*1.4 ,
+                # Number of packets sent = 100 * sampling rate of interface
+                self.assertTrue(100*0.6 <= data['flow_port_count'][ifindex] <= 100*1.4 ,
```

Comment:
Can you add spacing in between for better readability?

Reply:
Done.